src/parent/parent.js

```diff
@@ -729,7 +729,16 @@ export function parentComponent<P, X, C>({
         return clean.all(err);
       })
       .then(() => {
-        initPromise.asyncReject(err || new Error("Component destroyed"));
+        const error = err || new Error("Component destroyed");
+        if (
+          isElementClosed(currentContainer) ||
+          error.message === "Window navigated away"
```

Comment:
Can we create a top level variable for this error message? Seems like it's coming from here: https://github.com/krakenjs/zoid/blob/main/src/parent/parent.js#L826. It would be nice if both of these code paths shared the variable for this error message so if it changes in the future, this wouldn't break.